Source/LibationFileManager/InteropFactory.cs

```diff
@@ -52,94 +58,36 @@ static InteropFactory()
                 return;
             }
 
-            /*
-            * Commented code used to locate assemblies from the *ConfigApp.exe's module list.
-            * Use this method to locate dependencies when they are not in Libation's program files directory.
-#if DEBUG
-
-           // runs the exe and gets the exe's loaded modules
-           ModuleList = LoadModuleList(Path.GetFileNameWithoutExtension(configApp))
-               .OrderBy(x => x.ModuleName)
-               .ToList();
-#endif
-            */
-
             AppDomain.CurrentDomain.AssemblyResolve += CurrentDomain_AssemblyResolve;
 
             var configAppAssembly = Assembly.LoadFrom(configApp);
             var type = typeof(IInteropFunctions);
             InteropFunctionsType = configAppAssembly
                 .GetTypes()
-                .FirstOrDefault(t => type.IsAssignableFrom(t));
+                .FirstOrDefault(type.IsAssignableFrom);
         }
         private static string getOSConfigApp()
         {
-            var here = Path.GetDirectoryName(Environment.ProcessPath);
-
             // find '*ConfigApp.dll' files
             var appName =
-                Directory.EnumerateFiles(here, $"*{CONFIG_APP_ENDING}", SearchOption.TopDirectoryOnly)
-                // sanity check. shouldn't ever be true
-                .Except(new[] { Environment.ProcessPath })
+                Directory.EnumerateFiles(Configuration.ProcessDirectory, $"*{CONFIG_APP_ENDING}", enumerationOptions)
                 .FirstOrDefault(exe => MatchesOS(exe));
 
             return appName;
         }
 
-        /*
```

Comment:
Yeah, these were such hard-won knowledge that it feels weird to discard them wholesale. I'd like to keep the knowledge ... somewhere.

Reply:
Is keeping it in the demos sufficient?

Reply:
yup